dockers/tpu-tests/tpu_test_cases.jsonnet

```diff
@@ -14,7 +14,7 @@ local tputests = base.BaseTest {
   imageTag: std.extVar('image-tag'),
 
   tpuSettings+: {
-    softwareVersion: 'pytorch-nightly',
+    softwareVersion: 'pytorch-VERSION',
```

Comment:
@zcain117 I have replaced this one to use the same as installed XLA - to be aligned